velox/exec/RowContainer.h

```diff
@@ -302,6 +302,14 @@ class RowContainer {
       char* row,
       int32_t columnIndex);
 
+  /// Stores the first 'size' values from the 'decoded' vector into the
+  /// 'columnIndex' column of 'rows'.
+  void storeVector(
+      const DecodedVector& decoded,
+      const std::vector<char*>& rows,
```

Comment:
Can we use folly::Range here and remove 'size' parameter?

Can we rename this method to 'store'?

Reply:
Updated, thanks.